platforms/core-execution/build-cache-local/src/integTest/groovy/org/gradle/caching/local/internal/AbstractBuildCacheCleanupIntegrationTest.groovy

```diff
@@ -84,6 +85,7 @@ abstract class AbstractBuildCacheCleanupIntegrationTest extends AbstractIntegrat
         )
     }
 
+    @ToBeFixedForConfigurationCache(because = "Cache is cleaned twice on load after store")
```

Comment:
As mentioned in the description, the build cache is cleaned twice for a configuration cache miss build - once for the CC store operation and once for the CC load operation.